src/types/PageObject.ts

```diff
@@ -3,5 +3,5 @@ import type { Page } from '@playwright/test';
 export interface PageObject {
     readonly page: Page;
 
-    goTo(): Promise<void>;
+    goTo(...args: any): Promise<void>;
```

Comment:
You have to add the ignore of the eslint rule, if you want to use the `any` type here, but maybe there is a better solution.

This also isn't enough, because it will break the implementation of the actor `goesTo` method. We have to implement this as a general solution.